default.nix

```diff
@@ -251,31 +249,32 @@ in {
       find . -name '*.hs' -exec cp --parents {} $out \;
     '';
   };
-  binaries = {
-    Linux = hsPkgs.projectCross.musl64.hsPkgs.ormolu.components.exes.ormolu;
+  binaries = let hsPkgsOpt = hsPkgs.appendModule {
+    modules = [{
```

Comment:
What is the meaning of this manipulation? Does `appendModule` merge configurations or something?

Reply:
Yes, it merges (in the Nix module system sense) it with the arguments to `haskell-nix.project`. This is motivated by this (see PR description):

> Also, we now only set the GHC flags added in https://github.com/tweag/ormolu/pull/793 when actually building the executable, as otherwise the entire `ghc` library is rebuilt due to changed flags (which takes quite some time).